source/Scoreboard.cpp

```diff
@@ -0,0 +1,70 @@
+#include "../include/Scoreboard.hpp"
+
+std::ostream    &operator<<(std::ostream &stream, PlayerData &data)
+{
+  stream << data.playerName << ":" << data.playerClass << ":" << data.playerScore << " ";
+  return (stream);
+}
+
+PlayerData::PlayerData(std::string buf)
+{
+  std::istringstream        strstream(buf);
+  std::string               token;
+
+  isScore = true;
+  getline(strstream, token, ':');
+  playerName = token;
+  getline(strstream, token, ':');
+  playerClass = token;
+  getline(strstream, token, ':');
+  playerScore = atoi(token.data());
```

Comment:
atoi is a bit a messy function, use a function of the `stoi` family instead.
see: http://en.cppreference.com/w/cpp/string/basic_string/stol
Surround that by a `try {...} catch (...) [...}` and throw a more understandable exception.